### PR TITLE
Fix ledger split

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/processing/delta/ensure-value-consistency.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const ensureValueConsistency = (paymentRequest) => {
   // ensure no value gained or lost. if variation between total and lines apply difference to first gross line
   const invoiceLineTotal = getInvoiceLineTotal(paymentRequest)
-  const variation = Math.abs(paymentRequest.value) - Math.abs(invoiceLineTotal)
+  const variation = paymentRequest.value - invoiceLineTotal
```
Comment (johnwatson484) [inline app/processing/delta/ensure-value-consistency.js:4]: Should not be working with absolute values as will result in incorrect variance calculation

File: app/processing/delta/split-to-ledger.js [hunk 2]
```diff
@@ -3,16 +3,16 @@ const { AP } = require('../../ledgers')
 const ensureValueConsistency = require('./ensure-value-consistency')
 const { v4: uuidv4 } = require('uuid')
 
-const splitToLedger = (paymentRequest, unsettledValue, unsettledLedger) => {
+const splitToLedger = (paymentRequest, targetValue, ledger) => {
```
Comment (johnwatson484) [inline app/processing/delta/split-to-ledger.js:6]: renaming variables from `unsettled` as that is misleading when it comes to AR ledger

File: app/processing/delta/split-to-ledger.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ const copyPaymentRequest = (paymentRequest, ledger) => {
 
 const calculateInvoiceLines = (invoiceLines, apportionmentPercent) => {
   invoiceLines.map(x => {
-    x.value = x.value > 0 ? Math.ceil(x.value * apportionmentPercent) : Math.floor(x.value * apportionmentPercent)
+    x.value = Math.trunc(x.value * apportionmentPercent)
```
Comment (johnwatson484) [inline app/processing/delta/split-to-ledger.js:42]: We are working in pence so no need to do apply any rounding.  Can safely just drop decimals.  Applying rounding will result in incorrect line deltas

**Review by steve-dickinson — APPROVED** [review]
LGTM

**Pull request merged**